setup.py

```diff
@@ -36,10 +36,16 @@ def get_version(*file_paths):
         'xblock',
         'xblock.django',
         'xblock.reference',
+        'xblock.utils',
         'xblock.test',
         'xblock.test.django',
+        'xblock.test.utils',
     ],
     include_package_data=True,
+    package_data={
+        'xblock.utils': ['public/*', 'templates/*', 'templatetags/*'],
+        'xblock.test.utils': ['data/*'],
```

Comment:
(curious) Is this necessary? In the long run we don't want to include test data in the final package necessarily right?

Reply:
> In the long run we don't want to include test data in the final package necessarily right?

Perhaps not, In this `Xblock` repo we are running test cases on the pip-installed package rather than the project code
Here are the references:
https://github.com/openedx/XBlock/blob/master/tox.ini#L15-L17
https://github.com/openedx/XBlock/blob/master/setup.py#L39

If we don't add the test files in the package test cases don't run on them

Reply:
Got it, okay.